### PR TITLE
Add new columns to SHOW DATABASES docs

### PR DESCRIPTION
**Pull request by mnd999**
Add recently added columns creationTime, lastStartTime and lastStopTime
to SHOW DATABASES docs.

### CODE REVIEW

**Review by Hunterness — CHANGES_REQUESTED** [review]
Looks good so far but
- [x] there should be something in the `Additions, deprecations, removals` section about the new columns as well

**Comment by mnd999** [comment]
I don't think we've ever done that in the past just for adding columns to a the output.

**Comment by Hunterness** [comment]
I know I did it for the new SHOW TRANSACTIONS columns 🤷

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 1]
```diff
@@ -729,6 +729,22 @@ a|
...
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW DATABASES YIELD *
+----
+a|
+New columns have been added the result:
```
Comment (lidiazuin) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: This sentence is a bit confusing. Did you mean adding new columns to the result or as the result, for instance?

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 2]
```diff
@@ -729,6 +729,22 @@ a|
...
+
+* `creationTime`
+* `lastStartTime`
+* `lastStopTime`
+
+These columns are only returned in the full result set (with `YIELD`) and not by default.
```
Comment (lidiazuin) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc:747]:
```suggestion
These columns only return in the full result set (with `YIELD`) and not by default.
```

**Review by lidiazuin — CHANGES_REQUESTED** [review]
Thanks! Just a few comments from an editorial perspective.

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc (hunk 1 above)
Comment (mnd999) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: Yes, it is. I've added 'to'

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc (hunk 2 above)
Comment (mnd999) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc:747]: To me, it sounds more correct the way I had it.

**Review by lidiazuin — APPROVED** [review]

**Pull request merged**